clang/test/CIR/CodeGen/call.cpp

```diff
@@ -17,3 +17,15 @@ int f4() {
 // CHECK-LABEL: cir.func @f4() -> !s32i
 // CHECK:         %[[#x:]] = cir.call @f3() : () -> !s32i
 // CHECK-NEXT:    cir.store %[[#x]], %{{.+}} : !s32i, !cir.ptr<!s32i>
+
+int f5(int a, int *b, bool c);
```

Comment:
I noticed that if there is a definition of the function we're calling, tests like this would fail because we're generating a duplicate definition. Do you know why that's happening?